QueryProfiler/Profile/ProfileScheme.cs

```diff
@@ -7,18 +7,10 @@ namespace QueryProfiler
 {
    public class ProfileScheme
     {     // Operators
-        public int SearchCounter { get; set; }
-        public int ExtendCounter { get; set; }
-        public int JoinCounter { get; set; }
-        public int LookupCounter { get; set; }
-        public int ProjectCounter { get; set; }
-        public int ProjectAwayCounter { get; set; }
-        public int ProjectKeepCounter { get; set; }
         public int ContainsCounter { get; set; }
         public int Contains_csCounter { get; internal set; }
         public int Has_csCounter { get; internal set; }
         public int HasCounter { get; internal set; }

```

Comment:
please also add to the profileScheme the list of table names that were in the query